pkg/operator/tektonchains.go

```diff
@@ -64,7 +64,7 @@ func VerifySignature(resourceType string) {
 	resourceUID := cmd.MustSucceed("tkn", resourceType, "describe", "--last", "-o", "jsonpath='{.metadata.uid}'").Stdout()
 	resourceUID = strings.Trim(resourceUID, "'")
 	jsonpath := fmt.Sprintf("jsonpath=\"{.metadata.annotations.chains\\.tekton\\.dev/signature-%s-%s}\"", resourceType, resourceUID)
-	fmt.Println("Waiting 30 seconds")
+	log.Println("Waiting 30 seconds")
```

Comment:
why?

Reply:
SA1006: printf-style function with dynamic format string and no further arguments should use print-style function instead (staticcheck)

Reply:
@VeereshAradhya is it okay to log instead of printing? Because print is usually used for debugging and I get above error.

Reply:
Yes